src/legacy/ui/public/vislib/components/zero_injection/zero_filled_array.js

```diff
@@ -17,28 +17,25 @@
  * under the License.
  */
 
-export function VislibComponentsZeroInjectionZeroFilledArrayProvider() {
-  /*
-   * Accepts an array of x axis values (strings or numbers).
-   * Returns a zero filled array.
-  */
-
-  return function (arr, label) {
-    if (!Array.isArray(arr)) {
-      throw new Error('ZeroFilledArrayUtilService expects an array of strings or numbers');
-    }
+/*
+  * Accepts an array of x axis values (strings or numbers).
+  * Returns a zero filled array.
+*/
+export function createZeroFilledArray(arr, label) {
+  if (!Array.isArray(arr)) {
+    throw new Error('ZeroFilledArrayUtilService expects an array of strings or numbers');
```

Comment:
It's not a Service in the angular sense any more, so this error message could be updated

Reply:
For example, this text could change to: `new Error('createZeroFilledArray expects an array of strings or numbers');`

Reply:
solved in 0a9c70ab94